homeworks/roman.plynko_RPVanarx/homework_3/task2.js

```diff
@@ -0,0 +1,35 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+(function create() {
+    let searchPowNumber;
+    let minValue;
+    let maxValue;
+
+    function searchPow(number) {
```

Comment:
this function should not interact with outer variables, you script should contain only 2 elements:

1. recursive function taking one argument
2. the loop from min to max number loggin all number matching the condition